src/test/java/org/apache/commons/lang3/ObjectUtilsTest.java

```diff
@@ -148,11 +153,27 @@ public void testAnyNotNull() {
         assertTrue(ObjectUtils.anyNotNull(null, null, null, null, FOO, BAR));
     }
 
+    /**
+     * Tests {@link ObjectUtils#anyNull(Object...)}.
+     */
+    @Test
+    void testAnyNull() {
+        assertTrue(ObjectUtils.anyNull((Object) null));
+        assertTrue(ObjectUtils.anyNull(null, null, null));
+
+        assertFalse(ObjectUtils.anyNull());
+        assertFalse(ObjectUtils.anyNull((Object[]) null));
+        assertFalse(ObjectUtils.anyNull(FOO));
+        assertFalse(ObjectUtils.anyNull(null, FOO, null));
```

Comment:
Should this really be `assertFalse`?
`ObjectUtils.anyNull(null, null, null, null, FOO, BAR)` - should return `true`, because it has `null` elements in varargs.

Reply:
Thank you. You are right. I will create a new commit.